modular_skyrat/modules/customization/game/objects/items/plushes.dm

```diff
@@ -375,3 +375,19 @@
 	temp = starting_temp
 	calculate_height()
 	set_reagent_color_for_liquid()
+
+/obj/item/toy/plush/rubi
+	name = "huggable bee plushie"
+	desc = "It reminds you of a very, very, very huggable bee."
+	icon = 'modular_skyrat/master_files/icons/obj/plushes.dmi'
+	icon_state = "plushie_rubi"
+	gender = FEMALE
+	squeak_override = list('sound/weapons/thudswoosh.ogg' = 1)
+	attack_verb_continuous = list("hugs")
+	attack_verb_simple = list("hug")
+
+/obj/item/toy/plush/rubi/attack_self(mob/user)
+	. = ..()
+	user.changeNext_move(CLICK_CD_MELEE) // To avoid spam, in some cases (sadly not all of them)
+	SEND_SIGNAL(user, COMSIG_ADD_MOOD_EVENT, "hug", /datum/mood_event/warmhug, src)
```

Comment:
inb4 powergaming mood

Reply:
It's +1 mood for 2 minutes, you get more than that from petting pets in the Pet Zoo in the holodeck. And there's like, 15 of them ;)

Reply:
That being said, I feel like something should be done about the cooldown on that attack_self, but it'd have to be done for all plushies, because right now it's kind of silly how fast you can click.

Reply:
yeah maybe remove the hug event or add a +0 variant